.github/workflows/cd.yml

```diff
@@ -4,7 +4,7 @@ on: workflow_dispatch
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
why the change to 'latest'?

Reply:
The specific `ubuntu-20.04` tag turns out to be listed as `experimental`, whatever that means.